### DISCUSSION TITLE
Strip string from static resources

### DISCUSSION THREAD

**Discussion by robotfx** [FreshRSS/FreshRSS#3723] [Q&A] [answered]
I'm trying to use the static files without their strings, and I have noticed that not having the number string in the main.js link doesn't allow login. In other words, the <?= self::headScript() ?> generated the main.js link as 
`<script src="../scripts/main.js?1627148850" defer="defer" async="async"></script>
`
What I need is only
`<script src="../scripts/main.js" defer="defer" async="async"></script>
`
Is this possible? Except for the login issue, the site seems to function normally.
The .css doesn't cause any issue.

**Comment by Alkarex** [comment, marked as answer]
The number in the URL is mainly to avoid cache issues when `main.js` is updated on the server. If it does not work without it, try to clear your cache

**Comment by robotfx** [comment]
Got it. It was actually the bcrypt and maybe the extra.js. I've added them only to the login page, anf now it's fine.
